open_schools_platform/query_management/queries/selectors.py

```diff
@@ -40,3 +40,7 @@ def get_queries(*, filters=None) -> QuerySet:
     queries = QueryFilter(filters, qs).qs
 
     return queries
+
+
+def get_all_query_statuses() -> list:
+    return list(Query.Status.__dict__["_member_map_"].keys())
```

Comment:
selectors are user for extracting data from db, i would locate this func to Query model made it staticmethod